nuitka/specs/HardImportSpecs.py

```diff
@@ -144,3 +144,5 @@
     ),
     default_count=4,
 )
+
+ctypes_c_int_spec = BuiltinParameterSpec("ctypes.c_int", ("value",), default_count=1)
```

Comment:
Checking it, I saw this `class c_int(_ctypes._SimpleCData)` and the bases classes it has, maybe for the data value, we do not have to know very precisely in the class what kind of c types data it is, so `c_long` or not, can be inspected at compile time, if needed, but rather we might be able to treat all values the same, I mean for constant checks (probably `isinstance` needed there), and so on.

So I guess, you try and finish `c_int` and then we generalize it to cover all the types with the same data class from `replaceCompileTimeValue` generated, then things would be super easy.

As it stands, right now this only avoids the use of `ctypes.c_int` attribute lookups, not a whole lot.